### DISCUSSION TITLE
Added option to remove clicks/sec text in skin layout

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#23939] [Ideas]
In some high star stream maps, I often check clicks/sec in the auto mod of beatmap patterns to better understand how the finger works in those patterns, if possible, omit the clicks/sec line and only display numbers. Well, that would be great.
As far as I remember that HUD overlay is called ClicksPerSecondCounter.

https://github.com/ppy/osu/assets/128424762/a24fe107-bbec-4040-a0a0-457d0c251e1d

**Comment by integerrr** [comment]
First of all, please change the title of this discussion to English. 

Second, what's wrong with having the text "clicks/sec" showing in your (likely super niche) use case?

**Reply by huyenden** [reply]
Thanks for the reminder, I fixed it.
What I'm hoping for is that the number of clicks/sec only shows the number, although I can get used to having the line of clicks/sec all at once, but that seems like a pretty selfish request coming from me.
